### PR TITLE
Request Tests DAPP-403

### PR DESCRIPTION
**Pull request by berriose**
## Description

- update code to support testing of requests

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have added screenshots

### CODE REVIEW

**Review by ultimaterex — APPROVED** [review]

File: test/essential_test_provider_widget.dart [hunk 1]
```diff
@@ -37,14 +40,32 @@ Future<Widget> essentialTestProviderWidget({
     ...overrides,
   ];
 
+  // return ProviderScope(
```
Comment (KalervoHyyppa) [inline test/essential_test_provider_widget.dart:43]: Clean Up

**Review by KalervoHyyppa — CHANGES_REQUESTED** [review]
1 small comment

File: test/essential_test_provider_widget.dart (hunk 1 above)
Comment (berriose) [reply test/essential_test_provider_widget.dart:43]: Thanks for catching this. Updating now

**Review by lukorito — APPROVED** [review]

**Review by KalervoHyyppa — APPROVED** [review]
LGTM

**Pull request merged**